### PR TITLE
fix sorting

### PR DESCRIPTION
**Pull request by ux-AylonS**
Signed-off-by: AylonS <aylons@reblaze.com>

### CODE REVIEW

File: src/components/RbzTable.vue [hunk 1]
```diff
@@ -485,7 +485,7 @@ export default defineComponent({
       this.sortColumnTitle = column.title
 
       this.sortColumnDisplayFunction = (this.isSortByOriginalValue) ? null :
-        this.sortColumnDisplayFunction = column.displayFunction
+        column.displayFunction
```
Comment (Aviv-Galmidi) [inline src/components/RbzTable.vue:488]: That's not the right fix, the issue is with `this.isSortByOriginalValue`

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

File: src/components/RbzTable.vue [hunk 2]
```diff
@@ -485,7 +485,7 @@ export default defineComponent({
       this.sortColumnTitle = column.title
 
       this.sortColumnDisplayFunction = (this.isSortByOriginalValue) ? null :
```
Comment (Aviv-Galmidi) [inline src/components/RbzTable.vue:487]: The issue is with `this.isSortByOriginalValue`, should be `column.isSortByOriginalValue`

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

File: src/components/RbzTable.vue (hunk 2 above)
Comment (ux-AylonS) [reply src/components/RbzTable.vue:489]: ?

**Comment by Aviv-Galmidi** [comment]
Pulled this branch into mine and fixes will be included there

**Pull request closed without merging**